lms/resources/_js_config.py

```diff
@@ -140,6 +143,31 @@ def _add_canvas_submission_params(self, **kwargs):
         # Add the given document_url or canvas_file_id.
         self.config["submissionParams"].update(kwargs)
```

Comment:
Hard to see in the diff here, but `_add_canvas_submission_params` is out and about with the public methods. This should probably be shoved to the bottom.

Reply:
How did that happen? My precious alphabetical ordering. I'll fix it

Reply:
https://github.com/hypothesis/lms/pull/1547/commits/4c9d043f56a645d49ef4ed94da96c2726500a0a6